source/docs/prismic/content-slices.md

```diff
@@ -0,0 +1,262 @@
+
+## How to use Slices?
+
+Integrating a Prismic Slice Zone and its Slices into your project is a 3 steps process:
+
+<!-- Workaround for https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one#table-of-contents -->
+<!-- no toc -->
+1. [expose data in GraphQL](#Expose-data-in-GraphQL)
+2. [create a Slice library](#Create-a-Slice-library) related to these GraphQL types. A library is composed of pairs of React Components and GraphQL fragments.
+3. [update your static pages to retrieve the content dynamically](#Retrieve-and-display-content) and use the SliceZone component of your library to display the data appropriately
+
+## Expose data in GraphQL
+
+In this example, we will add a Slice Zone with 3 types of Slices to an existing home page.
+
+First, **update your GraphQL Schema** to reflect the Slice Zone definition. A Slice Zone will be designed as a [GraphQL union type](https://graphql.org/learn/schema/#union-types).
+
+```diff
+type Homepage {
+  some_text: String
+  title: String
+  sales_date: Date
++  mainContent: [HomePageSlice]
+}
++
++ union HomePageSlice = Carousel | Push | FeaturedProducts
++
++ type FeaturedProducts {
++   sectionTitle: String
++   category: Category
++ }
++ type Carousel {
++   slides: [CarouselSlide]
++ }
++ type Push {
++   blocks: [PushBlock]
++ }
+```
+
+In your resolvers, load your content as usual [with the Prismic loader](/docs/prismic/expose-content.html#Methods-to-query-content).
+
+In addition to the `fieldTransformers` parameter (for fields), the loader methods accept a `supportedSlices` key. It allows to define to data format for Slices supported by the content type, when a Slice Zone is configured.
+
+For a content type containing a title and a Slice Zone with only one type of Slice (`featured_products` exposed under the GraphQL `FeaturedProducts` type), the change would look like this:
+
+```diff
+{
+  fieldTransformers: {
+    title: new TitleTransformer(),
+  },
++  supportedSlices: [
++    new Slice("featured_products", { // Slice type as defined in Prismic
++      graphQLType: "FeaturedProducts", // type name as defined in your Schema
++      fieldTransformers: {
++        section_title: new TitleTransformer(),
++      },
++    }),
++    // ... add other Slice definitions here
++  ],
+};
+```
+
+The resolved content will now contain Slices contributed for the Slice Zone in the `content.body` field. Repeatable fields in a Slice are always made available under the `content.items` field of the Slice content.
+
+Here is an example showcasing how resolvers could allow to adapt Prismic content to the schema you've designed (and overcome the technical naming constraint brought by Prismic):
+
+```js
+// resolvers.js
+export default {
+  // […]
+  Homepage: {
+    mainContent: (content) => content.body,
+  },
+
+  Carousel: {
+    slides: (content) => content.items,
+  },
+  CarouselSlide: {
```

Comment:
and same remark here instead of CarouselSide resolver, I would put the PushBlock one to have an (almost fully) working example